app/src/main/java/cn/hospital/registerplatform/ui/component/hospital/DoctorDetailActivity.kt

```diff
@@ -64,11 +79,58 @@ class DoctorDetailActivity : ActionBarActivity("医生详情") {
             res.doSuccess {
                 mBinding.info = it
                 mBinding.executePendingBindings()
+                findViewById<AppCompatTextView>(R.id.comment_overview_score).text = it.averageRating.toString()
+                findViewById<AppCompatTextView>(R.id.comment_overview_people).text = it.commentsNum.toString() + "人"
             }
         }
-        hospitalViewModel.getDoctorScheduleList(doctorId).observe(this@DoctorDetailActivity) { res ->
-            res.doSuccess {
-                scheduleAdapter.updateList(it.sortedBy { info -> info.begin_time })
+        hospitalViewModel.getDoctorScheduleList(doctorId)
+            .observe(this@DoctorDetailActivity) { res ->
+                res.doSuccess {
+                    scheduleAdapter.updateList(it.sortedBy { info -> info.begin_time })
+                }
+            }
+
+
+        findViewById<Spinner>(R.id.user_comment_sort_method).onItemSelectedListener = object:
```

Comment:
since Binding object is generated, use Binding object to access View instead of findViewById